pages/events.jsx

```diff
@@ -64,7 +64,7 @@ var FormMailingListSignup = React.createClass({
           </fieldset>
           <fieldset>
             <label htmlFor="mailinglist-pp-note" className="sr-only">I'm okay with you handling this info as you explain in your <a href="https://www.mozilla.org/en-US/privacy/websites/">privacy policy</a></label>
-            <input id="mailinglist-pp-note" name={process.env.MAILINGLIST_PRIVACY_NAME} type="checkbox" checked readOnly required hidden />
+            <input id="mailinglist-pp-note" name={process.env.MAILINGLIST_PRIVACY_NAME} type="checkbox" className="sr-only" checked readOnly required />
```

Comment:
Nice, I like this solution, it actually shows up in NVDA (my screen reader) and is announced as read-only and required. Much more obvious than the current sign-up page on id.webmaker.org, where it's basically just treated as loose text by NVDA and could easily go unnoticed.